shqdrack.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Shqdrack",
+        "email": "shqdracky@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "shqdrack.github.io/Portfolio/"
```

Comment:
RECORD TYPE should be an actual record type. The value should be a hostname.

See docs: https://docs.is-a.dev